src/app.ts

```diff
@@ -236,11 +237,39 @@ export class App {
    *   ctx.res.set('X-Mojo', 'I <3 mojo.js!');
    *   await ctx.render(...args);
    * });
+   *
+   * // Render response with header using nested helper
+   * app.addHelper('renderWith.header', async (ctx, ...args) => {
+   *   ctx.res.set('X-Mojo', 'I <3 mojo.js!');
+   *   await ctx.render(...args);
+   * });
    */
   addHelper(name: string, fn: MojoAction): this {
-    return this.decorateContext(name, function (this: MojoContext, ...args: any[]) {
-      return fn(this, ...args);
-    });
+    const nestedNames = name.split('.');
+    if (nestedNames.length == 1)
```

Comment:
Not using blocks for anything else than a one-liner looks horrible.

Reply:
sure no problem